tool/tsh/mfa.go

```diff
@@ -663,3 +669,16 @@ func showOTPQRCode(k *otp.Key) (cleanup func(), retErr error) {
 		}
 	}, nil
 }
+
+func deleteTouchIDCredentialIfApplicable(credentialID string) {
+	err := touchid.AttemptDeleteNonInteractive(credentialID)
+	if err != nil {
+		if errors.Is(err, touchid.ErrCredentialNotFound) || errors.Is(err, touchid.ErrNotAvailable) {
+			// Nothing to do here, just proceed.
+			return
+		}
+		log.WithError(err).Errorf("Failed to delete credential: %s\n", credentialID)
+		return
+	}
+	fmt.Println("TouchID credential deleted.")
```

Comment:
My 2c: I wouldn't say anything here, it might be confusing for users to see an additional delete confirmation happen sometimes.